edx_django_utils/user/README.rst

```diff
@@ -0,0 +1,6 @@
+Django User and Group Management Code for Multiple Services
```

Comment:
This space would probably be more general. Does the following work for you? Also, the whole repo is for cross service utilities, so we can drop that from the title.
```suggestion
Django User and Group Utilities
```

Reply:
This comment is still relevant. ^

Reply:
@NIXKnight Please resolve this comment.

Reply:
I took the liberty of applying this change.